doc/source/serve/configure-serve-deployment.md

```diff
@@ -0,0 +1,25 @@
+(serve-configure-deployment)=
+
+# Configure a Ray Serve deployment
+
+These are all the properties available on a Ray Serve deployment. This is also available in the [API reference](../serve/api/doc/ray.serve.deployment_decorator.rst)
+
+The following can be configured through either the serve config file or on the `@serve.deployment` decorator:
```

Comment:
A few updates should be made to this content:

- Show a Python code sample and an equivalent config file.
- Show the usage of `.options()`.
- Explain the precedence (config > what's in the code).

I think we should also consolidate the content in the "app builder" page here to make it clear how to compose them. WDYT?

Reply:
Agreed I'll add this

Reply:
Added the 3 (actually moved them over from the serve config page because I think they're more relevant here). Also updated the serve config page. For the app builder, do you propose merging app builder content entirely into this page?